### PR TITLE
weathercom.py: Add option to specify locale.

### PR DESCRIPTION
**Pull request by facetoe**
It appears weather.com returns values in fahrenheit by default. This PR allows specifiying a locale that will cause weather.com to return values in units standard for that country (eg, en-AU==Celcius).

Potential fix for #577

### CODE REVIEW

**Comment by terminalmage** [comment]
This should already be handled by the ``units`` param. It defaults to metric, so we don't need to add a separate argument for this, we just need to handle the URL differently when units == metric.

**Comment by terminalmage** [comment]
Note that the Weather.com backend was recently re-done because Weather.com changed their API. You used to be able to just specify the units in the queryargs, That may no longer be possible though, so we could just pick a known metric locale and use that when units == metric.

**Comment by facetoe** [comment]
Yep, fair point. I guess we could just set the "locale" to en-AU when units==metric. That should return results in Celcius etc.

**Comment by terminalmage** [comment]
Yeah, that would do it.

When Weather.com changed their API, they removed the old JSON feed and the weather data is now fetched by some script and dumped into the page source as a blob of javascript. So I had to rewrite the module to parse the HTML instead.

I should have checked the metric units when I did the rewrite, shame on me for being so America-centric. :smile:

**Comment by facetoe** [comment]
Haha all good, I've changed it to determine the locale based on `units`.

File: i3pystatus/weather/weathercom.py [hunk 1]
```diff
@@ -123,6 +120,9 @@ class Weathercom(WeatherBackend):
     forecast page, the code you need will be everything after the last slash in
     the URL (e.g. ``94107:4:US``).
 
+    To receive values in units standard to your country, pass the relevant locale.
+    For example, locale='en_AU'.
+
```
Comment (terminalmage) [inline i3pystatus/weather/weathercom.py]: This no longer has to be here since the ``locale`` option has been removed.

Comment (facetoe) [inline i3pystatus/weather/weathercom.py]: Good catch!

**Comment by terminalmage** [comment]
LGTM

**Pull request merged**